src/VideoStreaming/VideoStreaming.pri

```diff
@@ -70,7 +70,6 @@ LinuxBuild {
 
         # Copy GStreamer plugins
         QMAKE_POST_LINK += $$escape_expand(\\n) xcopy \"$$GST_ROOT_WIN\\lib\\gstreamer-1.0\\*.dll\" \"$$DESTDIR_WIN\\gstreamer-plugins\\\" /Y $$escape_expand(\\n)
-        QMAKE_POST_LINK += $$escape_expand(\\n) xcopy \"$$GST_ROOT_WIN\\lib\\gstreamer-1.0\\validate\\*.dll\" \"$$DESTDIR_WIN\\gstreamer-plugins\\validate\\\" /Y $$escape_expand(\\n)
```

Comment:
@dogmaphobic These "validate" dlls don't exist in the 64 bit version. Is there a reason why they are needed? Things still compile fine.

Reply:
@jaxxzer Not sure if you may know?

Reply:
Oof, I guess they are not needed, but I really have no clue. To my knowledge, we aren't using it.